### ISSUE TITLE
需要离线部署该项目，如何提前把模型下载好？

### ISSUE THREAD

**Issue by luxinfeng** [InternLM/magic-doc#31]
我们的场景下需要离线部署该项目，但是实际测试中发现需要ocr识别的时候，该项目需要联网下载模型，这个模型可以提前下载好吗？下载好后放到什么位置呢

**Comment by icecraft** [comment]
yes, you can prepare the models manually. it is under `$HOME/.paddleocr` on linux platform.
